examples/BuddyLlama/llama-main.cpp

```diff
@@ -36,45 +37,44 @@ int main() {
   string pureStr;
   cout << "Please enter what you want to say to me" << endl;
   getline(cin, pureStr);
-  clock_t buddyTokenizeStart = clock();
+  auto buddyTokenizeStart = system_clock::now();
   Text<size_t, 2> pureStrContainer(pureStr);
   pureStrContainer.tokenizeLlama(vocabDir, 80);
-  clock_t buddyTokenizeEnd = clock();
-  double buddyTokenizeTime =
-      (double)(buddyTokenizeEnd - buddyTokenizeStart) / CLOCKS_PER_SEC;
+  auto buddyTokenizeEnd = system_clock::now();
+  auto buddyTokenizeTime = duration_cast<seconds>(buddyTokenizeEnd - buddyTokenizeStart);
   // Print the tokenized result
   cout << "Get User input:" << pureStrContainer.revert(pureStrContainer)
        << endl;
-  cout << "[Buddy] Tokenize input time: " << buddyTokenizeTime * 1000 << "ms"
+  cout << "[Buddy] Tokenize input time: " << buddyTokenizeTime.count() * 1000 << "ms"
```

Comment:
What is the unit of time measurement here?
The unit multiplied by 1000 here is “ms”, and the unit divided by 1000 below is “s”.

Reply:
Yes! I have change buddyTokenizeTime's time unit to milliseconds and update this cout.